src/interfaces.ts

```diff
@@ -1,53 +1,70 @@
-import { IBackoffOpts } from 'backoff-web';
-
-export interface IServerOpts {
-  accessToken: string,  // genesyscloud auth token
-  environment: string,  // genesyscloud api environment
-  appVersion: string,   // version of app using the logging library
-  logTopic: string,     // all local logs will be prefixed by this
-  logLevel: LogLevels,  // logs at this level or high get sent to the server
-  uploadDebounceTime?: number,   // time to debounce logs uploads to the server
+export interface ILoggerConfig {
+  /** JWT access token to use in HTTP request */
+  accessToken: string;
+  /**
+   * url to send the logs to (note this needs to be the full URL)
+   * an HTTP `POST` request will be issued to this url
+   */
+  url: string;
+  /** version of app using the logging library */
+  appVersion: string;
+  /** all local logs will be prefixed by this. */
+  logTopic: string;
+  /** initialize server logging. defaults to `true` */
+  initializeServerLogging?: boolean;
```

Comment:
Added this to skip setting up the server-logging (don't know why you would want to do that, but w/e)